test/js/download.test.js

```diff
@@ -28,6 +47,11 @@ describe('Download components for faceted search', () => {
       .mockImplementation(() => {
         return studySearchContext
       })
+
```

Comment:
Let's talk about this next week -- I think we can use a different pattern that doesn't require either the spyOn or the extra useContextDownload method